tests/wpt/tests/html/canvas/offscreen/manual/the-offscreen-canvas/offscreencanvas.transfercontrol.to.offscreen.nocrash.html

```diff
@@ -0,0 +1,12 @@
+<title>Transfer Canvas without Context</title>
```

Comment:
Instead of making this a testharness test (since we don't make any assertions about the results), we can still make this a crashtest: https://web-platform-tests.org/writing-tests/crashtest.html#crashtest-tests . It will require renaming the test file.